src/lib/babel-config.js

```diff
@@ -4,12 +4,8 @@ export default (env, options={}) => ({
 		[require.resolve('babel-preset-env'), {
 			loose: true,
 			modules: options.modules || false,
+			browsers: options.browsers,
```

Comment:
Oh yeah, nice catch! Looks like that should have always been `target.browsers` 👍 

Sorry, I'm confused by your first sentence. Not sure what you mean.